### PR TITLE
Using consul image from Dockerhub

### PR DESCRIPTION
**Pull request by markusthoemmes**
- Bumping consul version to 0.7.0
- simplifying configuration file (to be resurrected later when actually building a cluster)
- adjusted redo
- removed own Dockerfile

### CODE REVIEW

File: ansible/roles/consul/tasks/deploy.yml [hunk 1]
```diff
@@ -24,15 +20,14 @@
 - name: (re)start consul server/agent
   docker:
     name: consul
-    image: "{{ docker_registry }}{{ docker_image_prefix }}/consul:{{ docker_image_tag }}"
+    image: consul
```
Comment (domdom82) [inline ansible/roles/consul/tasks/deploy.yml]: We should pin this.

File: ansible/roles/consul/templates/config.json.j2 [hunk 2]
```diff
@@ -1,32 +1,14 @@
-{# this template is used to generate a config.json for booting a consul server or a consul agent, based on the given environment inventory #}
```
Comment (domdom82) [inline ansible/roles/consul/templates/config.json.j2]: We will resurrect this from history later.

Soon I hope.

**Comment by domdom82** [comment]
travis got timeout. I restarted.

File: ansible/roles/consul/tasks/deploy.yml (hunk 1 above)
Comment (markusthoemmes) [inline ansible/roles/consul/tasks/deploy.yml]: Good one, was lost in some amending. Thanks.

**Comment by markusthoemmes** [comment]
@domdom82 added 2 more fixes, kept them seperately for your reviewing pleasure. Those are to be squashed.

**Comment by rabbah** [comment]
image is ~93% smaller :+1:

**Comment by domdom82** [comment]
LGTM - can we safely use that button?

**Comment by rabbah** [comment]
waiting on @bjustin-ibm to give a :+1:.
is there a pg?

**Comment by markusthoemmes** [comment]
PG2 178 approved

**Comment by markusthoemmes** [comment]
PG1 470,471,475,476,477
PG2 176,178,191

All approved this change. Cannot do more to verify. The intermittent failure i've seen was not related to the consul change.

**Comment by domdom82** [comment]
green PG 485

**Pull request merged**